### PR TITLE
Vier dark item links

### PR DESCRIPTION
**Pull request by rebeka-catalina**
Make links visually different from normal text

### CODE REVIEW

**Comment by annando** [comment]
Would be define this as bugfix?

**Comment by tobiasd** [comment]
Yes I think it is a bugfix, as not seeing a link due design qualifies as a design flaw for me.

File: view/theme/vier/dark.css [hunk 1]
```diff
@@ -8,6 +8,9 @@ hr { background-color: #343434 !important; }
 a, .wall-item-name, .fakelink { 
 	color: #989898 !important; 
 }
+.wall-item-content a {
+	        font-weight: bold;
```
Comment (Hypolite) [inline view/theme/vier/dark.css]: The indentation is all sorts of wrong ☹️

Comment (tobiasd) [inline view/theme/vier/dark.css]: I think changing it would be better then fetching up a PR that is 2 month old. ;-)

**Pull request merged**